src/scripts/browserDeviceProfile.js

```diff
@@ -233,8 +239,20 @@ function supportsHlg(options) {
 }
 
 function supportsDolbyVision(options) {
+    // Safari 13.1 added support for Dolby Vision:
+    //  * https://developer.apple.com/documentation/safari-release-notes/safari-13_1-release_notes
+    const safariSupportsDoVi = browser.safari && (false // eslint-disable-line sonarjs/no-redundant-boolean
+        || browser.versionMajor == 13 && browser.versionMinor >= 1
+        || browser.versionMajor >= 14
+    );
+
     return options.supportsDolbyVision ?? (false // eslint-disable-line sonarjs/no-redundant-boolean
-            || browser.safari && ((browser.iOS && browser.iOSVersion >= 13) || browser.osx)
+            || safariSupportsDoVi
+            // Chrome/Chromium-based browsers added support for Dolby Vision fallback on version 122+.
+            // This does not mean its supported, just that media extensions now report support (and more importantly, no support) properly.
+            //   * https://chromium-review.googlesource.com/c/chromium/src/+/5058372
+            || browser.chrome && (browser.versionMajor >= 122)
+            || browser.edgeChromium && (browser.versionMajor >= 122)
```

Comment:
I'm not sure about Chromium on Apple, maybe @gnattu can give more insights about that. But [HLS.js](https://github.com/video-dev/hls.js) used on Windows/Linux does not yet support `dvh1/dvhe` fourcc. Even edgeChromium requires an additional plugin and an experimental flag to enable DoVi support.

So claiming Chromium supports DoVi does no good, we already use their fallback layer to play HDR/HLG/SDR.

Reply:
The goal here wasn't to "enable" DoVi for these browsers but to indicate that these browsers now support accurate reporting of their capabilities. See the linked Chromium CL comment for info and the bug number.

It looks like the Chromium team is staging changes to support Dolby Vision on different platforms ([macOS CL](https://chromium-review.googlesource.com/c/chromium/src/+/5667325)), so enabling this should automatically enable DoVi once that lands, no?

HLS.js not supporting might be the bigger issue here for other platforms.

Reply:
Please don't. It is still lying about that support and will report it 'maybe' plays dovi profile 5 videos when it will do client side tone mapping with wrong colorspace